src/trace_processor/importers/proto/winscope/winscope_rect.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SRC_TRACE_PROCESSOR_IMPORTERS_PROTO_WINSCOPE_WINSCOPE_RECT_H_
+#define SRC_TRACE_PROCESSOR_IMPORTERS_PROTO_WINSCOPE_WINSCOPE_RECT_H_
+
+#include <algorithm>
+#include "protos/perfetto/trace/android/graphics/rect.pbzero.h"
+#include "protos/perfetto/trace/android/surfaceflinger_layers.pbzero.h"
+
+namespace perfetto {
+namespace trace_processor {
+
+struct Point {
+  double x;
+  double y;
+};
+
+struct Size {
+  double w;
+  double h;
+};
+
+struct WinscopeRect {
+  double x = 0;
+  double y = 0;
+  double w = 0;
+  double h = 0;
+
+  static WinscopeRect makeRect(double left,
```

Comment:
Please folllow the C++ style guidelines (captial functions, avoid unnecessary statics functions when normal constructors will do etc).